Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:20-alpine
+
+RUN npm install -g nodemon
```

Comment:
Shouldn't we use Yarn here?

Reply:
Yes removing `RUN npm install -g nodemon` works perfectly fine.